docs/building-modules/errors.md

```diff
@@ -30,17 +30,17 @@ necessarily have to be. The only restrictions on error codes are the following:
 
 Note, the SDK provides a core set of *common* errors. These errors are defined in `types/errors/errors.go`.
 
-## Wrapping
```

Comment:
imo we should leave this section and add a small note that users can wrap the errors using the standard `errors` pkg.

Reply:
Good one, will revert this. Thanks!